torch/csrc/autograd/python_variable.cpp

```diff
@@ -222,7 +222,7 @@ PyObject * THPVariable_Wrap(at::TensorBase var)
     }
   }
 
-  if (C10_LIKELY(var.device().type() != c10::kXLA)) {
+  if (C10_LIKELY(var.device().type() != c10::kXLA && var.device().type() != c10::kLazy)) {
```

Comment:
we can remove this part for now since it's also needed if we were to use Python subclass extending torch.tensor.